CRM/Core/BAO/SchemaHandler.php

```diff
@@ -221,10 +222,8 @@ public static function buildIndexSQL(&$params, $separator) {
   /**
    * @param string $tableName
    * @param string $fkTableName
-   *
-   * @return bool
    */
-  public static function changeFKConstraint($tableName, $fkTableName) {
+  public static function changeFKConstraint($tableName, $fkTableName): void {
```

Comment:
@aydun all these deep unanswered philosophical questions leave me with a vague feeling of emptiness.